Source/WebKit/UIProcess/Launcher/ProcessLauncher.h

```diff
@@ -45,7 +45,10 @@
 #endif
 
 #if USE(EXTENSIONKIT)
-OBJC_CLASS _SEExtensionProcess;
+OBJC_CLASS BEWebContentProcess;
+OBJC_CLASS BENetworkingProcess;
+OBJC_CLASS BERenderingProcess;
+using ExtensionProcess = std::variant<RetainPtr<BEWebContentProcess>, RetainPtr<BENetworkingProcess>, RetainPtr<BERenderingProcess>>;
```

Comment:
To represent an "invalid" / "empty" ExtensionProcess, you could:
1. use `std::optional<ExtensionProcess>`
2. Add a `std::monostate` as first alternative: `std::variant<std::monostate, RetainPtr<BEWebContentProcess>, RetainPtr<BENetworkingProcess>, RetainPtr<BERenderingProcess>>`
3. Have your member functions return a `ExtensionProcess*` and nullptr in case of error.